src/web/mjs/connectors/Qiman5.mjs

```diff
@@ -53,6 +53,19 @@ export default class Qiman5 extends SinMH {
         `;
     }
 
+    canHandleURI(uri) {
+        super.initialize()
+            .then(() => {
```

Comment:
This does not work as intended, `canHandleURI` is expected to return a `bool`, but the current implementation returns a `Promise<bool>`, therefore it will always be true even if the ``promise resolves to false.

Reply:
Hum, but we must initialize the connector before otherwise copypaste wont work. I cant call initialize with await without async and i shouldnt prefix canHandleURI with async, right? 

Btw its a common occurrence when connector have to assign an url and first thing user do is pasting.
What should i do ?